scania-digital-design-system/sdds
# fix(data-table): migration to tegel

**Describe pull-request**  
Migration of data-table component

**Solving issue**  
Fixes: [AB#2455](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2455)

**How to test**  
1. Open the link from Netlify Storybook
2. Click around data-table and see if all stories render properly. There should be 10 stories in total including the native one.


**Additional context**  
Be aware that the data-table consists of 9 subcomponents so theming is not done in this PR as it involves a lot of work. In order to proceed with work on the data-table, just a basic migration of controls and readme files was done. other segments of migration will be done in separate PRs.
